ingest-app/docs/api.md

```diff
@@ -768,7 +1026,11 @@ Granule metadata can be translated between metadata standards using the translat
 Example: Translate ECHO10 metadata to UMM-G
 
 ```
-curl -i -XPOST -H "Content-Type: application/echo10+xml" -H "Accept: application/vnd.nasa.cmr.umm+json;version=1.4" %CMR-ENDPOINT%/translate/granule -d \
+curl -i -XPOST \
+  -H "Content-Type: application/echo10+xml" \
+  -H "Accept: application/vnd.nasa.cmr.umm+json;version=1.5" \
```

Comment:
I think I'm wrong, 1.6 is the latest, correct?

Reply:
It can be any version that we support.

Reply:
I'm going to move it to 1.6, the idea is that we don't want to encourage users to run older versions unless they need to.